src/cpptests/t_forkgc.cpp

```diff
@@ -14,6 +14,32 @@ static timespec getTimespecCb(void *) {
   return ts;
 }
 
+typedef struct {
+  RedisModuleCtx *ctx;
```

Comment:
RSDummyContext

Reply:
This is using the mock API.
In line 68: `RMCK::Context ctx;`